app/scripts/modules/kubernetes/src/v2/pipelines/stages/patchManifest/PatchManifestOptionsForm.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { CheckboxInput, StageConfigField } from '@spinnaker/core';
+import Select, { Option } from 'react-select';
+
+export interface IPatchManifestOptionsFormProps {
+  record: boolean;
+  onRecordChange: (_: boolean) => void;
```

Comment:
I could be wrong here but I thought underscores denoted unused params, maybe we could name these to avoid ambiguity?